provider/maas/environ.go

```diff
@@ -337,6 +337,11 @@ func (env *maasEnviron) SupportsSpaces(ctx context.ProviderCallContext) (bool, e
 	return true, nil
 }
 
+// SupportsProviderSpaces is specified on environs.Networking.
+func (env *maasEnviron) SupportsProviderSpaces(ctx context.ProviderCallContext) (bool, error) {
+	return false, nil
```

Comment:
This and AWS are the wrong way around. MAAS does support provider spaces; AWS does not.